src/components/Reviews.jsx

```diff
@@ -49,6 +64,7 @@ function Reviews() {
         </div>
         </Link>
             )
+            }
```

Comment:
I like that you've made the whole card a link here, and the hover CSS here is really effective.